### PR TITLE
`sin()` - New CSS functional notation

### PR DESCRIPTION
**Pull request by ramiy**
#### Summary
Document the CSS `sin()` functional notation.

#### Motivation
[Safari/webkit already supports](https://webkit.org/blog/12445/new-webkit-features-in-safari-15-4/) this CSS function. [Chromium](https://bugs.chromium.org/p/chromium/issues/detail?id=1190444) will follow soon.

#### Supporting details
https://drafts.csswg.org/css-values/#trig-funcs

#### Metadata
- [x] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by ramiy** [comment]
The `bad_bcd_queries: No BCD data for query: css.types.sin` flaw will be fixed with https://github.com/mdn/browser-compat-data/pull/17170 merge.

File: files/en-us/web/css/sin/index.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+div {
+  background-color: red;
+  width: calc( sin(90deg) * 100px );
+  height: calc( sin(90deg) * 100px );
+}
+```
+
+### Controling Animation Duration
+
+Another usecase is to control the {{cssxref("animation-duration")}}. Redusing duration based on the sine value. In this case the animation duration will be `1s`.
```
Comment (estelle) [inline files/en-us/web/css/sin/index.md:48]:
```suggestion
Another use-case is to control the {{cssxref("animation-duration")}}. Reducing duration based on the sine value. In this case, the animation duration will be `1s`.
```

File: files/en-us/web/css/sin/index.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+  - Layout
+  - Reference
+  - Web
+  - sin
+browser-compat: css.types.sin
+spec-urls: https://drafts.csswg.org/css-values/#trig-funcs
+---
+{{CSSRef}}
+
+The **`sin()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS_Functions) is a trigonometric function that returns the sine of a number, which is a value between `-1` and `1`. The function contain a single calculation which must resolve to either a `<number>` or an `<angle>` by interpreting the result of the argument as radians. That is, `sin(45deg)`, `sin(0.125turn)`, and `sin(3.14159 / 4)` all represent the same value, approximately `.707`.
```
Comment (estelle) [inline files/en-us/web/css/sin/index.md:18]:
```suggestion
The **`sin()`** [CSS](/en-US/docs/Web/CSS) [function](/en-US/docs/Web/CSS/CSS_Functions) is a trigonometric function that returns the sine of a number, which is a value between `-1` and `1`. The function contains a single calculation that must resolve to either a `<number>` or an `<angle>` by interpreting the result of the argument as radians. That is, `sin(45deg)`, `sin(0.125turn)`, and `sin(3.14159 / 4)` all represent the same value, approximately `.707`.
```

File: files/en-us/web/css/sin/index.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+### Formal syntax
+
+{{CSSSyntax}}
+
+## Examples
+
+### Boxes Size
+
+For example, when creating a 100x100 box based on external parameter, in this case `sin(90deg)`. Thus `sin(90deg)` will return `1` making the box `100px` width and `100px` height.
```
Comment (estelle) [inline files/en-us/web/css/sin/index.md:36]:
```suggestion
For example, when creating a 100x100 box based on external parameters, in this case `sin(90deg)`. Thus `sin(90deg)` will return `1` making the box `100px` width and `100px` height.
```

File: files/en-us/web/css/sin/index.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+}
+```
+
+### Controling Animation Duration
+
+Another usecase is to control the {{cssxref("animation-duration")}}. Redusing duration based on the sine value. In this case the animation duration will be `1s`.
+
+```css
+div {
+  animation-name: MyAnimation;
```
Comment (estelle) [inline files/en-us/web/css/sin/index.md:52]:
```suggestion
  animation-name: myAnimation;
```

**Review by estelle — APPROVED** [review]
Awesome! Thank you for adding this.

**Pull request merged**